### DISCUSSION TITLE
Field lookup option: return country name instead of lon/lat

### DISCUSSION THREAD

**Discussion by peterdk** [grafana/grafana#43964] [Dashboards, Panels & Visualizations]
I have a data source with country codes (US,NL,KO, etc.) and I want to display them nicely in a table. I can't find a way to easily display country names, but did find the Field lookup transform option. It detects the country codes, but returns the lat lon, while I need the country name. 

The plugin is in alpha I understand, so I would like to request that it gets the option to return other fields like the country name. It seems the data is there.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
